src/components/StudioTopbar/StudioTopbar.style.tsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Link } from 'react-router-dom'
+import { breakpoints, colors, sizes, transitions, typography, zIndex } from '@/shared/theme'
+import { TopbarBase, Avatar, Text, Button, TOP_NAVBAR_HEIGHT, TopbarBaseProps } from '@/shared/components'
+
+type CommonStudioTopbarProps = {
+  isActive?: boolean
+}
+
+export const StyledTopbarBase = styled((props: TopbarBaseProps) => <TopbarBase {...props} />)`
+  @media screen and (min-width: ${breakpoints.small}) {
+    display: flex;
+    justify-content: space-between;
+  }
+`
+
+export const DrawerButton = styled(Button)<CommonStudioTopbarProps>`
+  svg {
+    transform: rotate(${({ isActive }) => (isActive ? '180deg' : '0')});
+    transition: transform ${transitions.timings.regular} ${transitions.easing};
+  }
+`
+
+export const ChannelInfoContainer = styled.div<CommonStudioTopbarProps>`
+  display: flex;
+  align-items: center;
+  background-color: ${({ isActive }) => isActive && colors.gray[600]};
+  &:hover {
+    cursor: pointer;
+    background-color: ${({ isActive }) => (isActive ? colors.gray[600] : colors.gray[700])};
+  }
+  svg {
+    margin-left: auto;
+    margin-right: ${sizes(3)};
+  }
+`
+
+export const StyledAvatar = styled(Avatar)`
+  margin-left: ${sizes(5)};
+  margin-right: ${sizes(2.5)};
+`
+
+export const TextContainer = styled.div`
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 160px;
+  p:nth-of-type(1) {
+    font-size: ${typography.sizes.body1};
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+  }
+  p:nth-of-type(2) {
+    font-size: ${typography.sizes.caption};
+    color: ${colors.gray[300]};
+  }
+`
+
+export const StudioContainer = styled.div`
+  display: flex;
+  align-items: center;
+  margin-left: auto;
+  margin-right: ${sizes(3)};
+  svg {
+    width: auto;
+    height: auto;
+  }
+  button {
+    cursor: pointer;
+  }
+  ${ChannelInfoContainer} {
+    &:hover {
+      background-color: ${colors.transparent};
+    }
+    ${TextContainer} {
+      display: none;
+      @media screen and (min-width: ${breakpoints.medium}) {
+        display: flex;
+      }
+    }
+  }
+`
+
+export const AddVideoButton = styled(Button)`
+  display: none;
+  @media screen and (min-width: ${breakpoints.small}) {
+    display: flex;
+  }
+`
+
+export const MemberInfoContainer = styled.div`
+  display: flex;
+  align-items: center;
+  margin: calc(-1 * ${sizes(3)});
+  margin-bottom: 0;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.18);
```

Comment:
Let's try to use one of `colors.transparentWhite` for this. The closest we have defined is `32`, not sure if it will work, if not we can just define `18` there